packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -686,6 +687,9 @@ export function authorizationServiceBuilder(
       if (jwk.kty !== "RSA") {
         throw invalidKey(keySeed.key, "Not an RSA key");
       }
+
+      assertValidateRsaKeyLength(jwk);
```

Comment:
See comment below - if you accept my suggestion, you won't have to call this here, it will be done inside `createJWK`